src/static/js/demo.js

```diff
@@ -96,9 +99,53 @@ function configureControlPanel() {
   }
 }
 
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function preventZoomingPage() {
+  document.addEventListener(
+    'wheel',
+    e => {
+      if (e.ctrlKey) event.preventDefault(); //prevent zoom
+    },
+    { passive: false, capture: 'bubble' },
+  );
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+function setupFixedDiagramContainerSize(containerId) {
+  const containerElt = document.getElementById(containerId);
+  const height = containerElt.parentNode.parentNode.getBoundingClientRect().height;
+  // parent height minus 2 x padding
+  containerElt.style = `overflow: hidden; height:${height - 2 * 20}px`;
+}
+
 // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
 function startDemo() {
-  startBpmnVisualization({ container: 'bpmn-container' });
+  preventZoomingPage();
+  const bpmnContainerId = 'bpmn-container';
+  setupFixedDiagramContainerSize(bpmnContainerId);
+
+  const parameters = new URLSearchParams(window.location.search);
+  const zoomThrottleElt = document.getElementById('zoom-throttle'),
+    zoomDebounceElt = document.getElementById('zoom-debounce'),
+    zoomControlsElt = document.getElementById('zoom-config-controls');
```

Comment:
:question: I currently don't see this in the page, is that the expected behaviour?

Reply:
this was an issue on Firefox, that is now fixed with a8c6e39

Reply:
yes, this is in purpose as it works only on instantiation, at this moment no implementation to change it in runtime